web/src/components/molecules/CreatedDateRangeFilter/hooks.ts

```diff
@@ -0,0 +1,22 @@
+import { Moment } from 'moment';
+import { useState } from 'react';
+
+export const useHooks = () => {
+  const [startDate, setStartDate] = useState<Moment | null>();
+  const [endDate, setEndDate] = useState<Moment | null>();
+
+  const handleStartDateChange = (date: Moment | null): void => {
+    setStartDate(date);
+  };
+
+  const handleEndDateChange = (date: Moment | null): void => {
```

Comment:
@jeremiahC Yes sir, there would be a type mismatch without the null as seen in this error message.
![image](https://github.com/framgia/sph-sim-jms/assets/116238730/f1315815-e407-4297-940d-af5c0c88fe61)

It's also possible to clear the date fields (i.e. when we no longer want to filter jobs by date) using `null` as the on change value. I'll include that change here since I implemented it in the FE task.